locale/af/LC_MESSAGES/server.po

```diff
@@ -240,29 +240,206 @@ msgstr ""
+msgid "Welcome!"
+msgstr ""
+
+#: templates/verify.html:3 templates/verify.txt:2
+msgid "Please activate your account by confirming this email address."
+msgstr ""
+
+#: templates/verify.html:4 templates/verification_reminder.html:4
+msgid "Activate now"
+msgstr ""
+
+#: templates/verify.html:5 templates/post_verify.html:5 templates/reset.html:5
+#: templates/unlock.html:5 templates/verification_reminder.html:5
+msgid "Alternatively:"
+msgstr ""
+
+#: templates/verify.html:7
+msgid ""
+"For more information, please visit <a href='https://support.mozilla.org' "
+"style='color: #0095dd; text-decoration: none;'>Mozilla Support</a>"
+msgstr ""
+
+#: templates/verify.html:8 templates/verification_reminder.html:8
+msgid "Verify Email"
+msgstr ""
+
+#: templates/verify.html:9 templates/verification_reminder.html:9
+msgid "Verify your email to finish your Firefox Account registration"
+msgstr ""
+
+#: templates/verify.txt:3
+msgid "Activate now:"
+msgstr ""
+
+#: mailer.js:115 mailer.js:120 mailer.js:304
 msgid "Verify your Firefox Account"
 msgstr ""
 
-#: mailer.js:121
+#: mailer.js:145 mailer.js:151
 msgid "Reset your Firefox Account password"
 msgstr ""
 
-#: mailer.js:155
+#: mailer.js:176 mailer.js:183
 msgid "Re-verify your Firefox Account"
 msgstr ""
 
-#: mailer.js:185
-msgid "Your Firefox Account password has been changed"
+#: templates/_pending.txt:1
+msgid "Still there?"
 msgstr ""
 
-#: mailer.js:212
-msgid "Your Firefox Account password has been reset"
+#: templates/_pending.txt:2
+msgid ""
+"A while ago you created a Firefox Account, but never verified it. Click the "
+"link below to activate your account."
 msgstr ""
 
-#: mailer.js:239
-msgid "A new device is now syncing to your Firefox Account"
+#: templates/_pending.txt:3 templates/post_verify.html:3
+msgid ""
+"You successfully connected your first device to your Firefox Account. Now "
+"you can sign in to Sync from your other devices, including Firefox for <a "
+"href='%(androidUrl)s'>Android</a> and <a href='%(iosUrl)s'>iOS</a>."
+msgstr ""
+
+#: templates/_pending.txt:4
+msgid ""
+"You successfully connected your first device to your Firefox Account. To "
+"make the most of Sync, connect any <a href='%(androidUrl)s'>Firefox for "
+"Android</a> or <a href='%(iosUrl)s'>Firefox for iOS</a>."
+msgstr ""
+
+#: templates/_pending.txt:5 templates/post_verify.html:4
+msgid "Connect another device"
+msgstr ""
+
+#: templates/_pending.txt:6
+msgid "Sign in to Sync"
+msgstr ""
+
+#: templates/new_sync_device.html:3 templates/new_sync_device.txt:2
+msgid ""
+"A device began syncing with the devices connected to your Firefox Account."
+msgstr ""
+
+#: templates/new_sync_device.html:5 templates/password_changed.html:5
+#: templates/password_reset.html:5 templates/post_verify.html:7
+#: templates/reset.html:7 templates/unlock.html:7
+#: templates/verification_reminder.html:7 templates/verify.html:7
+msgid ""
+"For more information, please visit <a href='%(supportUrl)s' style='color: "
```

Comment:
The intent was to merge just after train 47 was cut and 48 started. Which git sha of the auth-mailer made train 47? If the sha includes the new stings, then I'll re-cut the strings.